packages/tree-sitter-lua/build.sh

```diff
@@ -0,0 +1,21 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/tree-sitter-grammars/tree-sitter-lua
+TERMUX_PKG_DESCRIPTION="Lua grammar for tree-sitter"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.1.0"
+TERMUX_PKG_SRCURL=https://github.com/tree-sitter-grammars/tree-sitter-lua/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=230cfcbfa74ed1f7b8149e9a1f34c2efc4c589a71fe0f5dc8560622f8020d722
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_EXTRA_MAKE_ARGS="
+PARSER_URL="${TERMUX_PKG_HOMEPAGE}"
+"
+
+termux_step_pre_configure() {
+	rm setup.py pyproject.toml
+}
+
+termux_step_post_make_install() {
+	install -d "${TERMUX_PREFIX}"/share/nvim/runtime/parser/
+	ln -s "${TERMUX_PREFIX}"/lib/libtree-sitter-lua.so "${TERMUX_PREFIX}"/share/nvim/runtime/parser/lua.so
```

Comment:
> [!IMPORTANT]
> It should probably be the `neovim` package's responsibility to link the `tree-sitter-${filetype}` parsers into its runtime.
>
> #### *The parsers should exist independently of Neovim.*

<details><summary>Here is the contents of Arch's `tree-sitter-lua` package.</summary>
<p>

```styl
tree-sitter-lua /usr/
tree-sitter-lua /usr/include/
tree-sitter-lua /usr/include/tree_sitter/
tree-sitter-lua /usr/include/tree_sitter/tree-sitter-lua.h
tree-sitter-lua /usr/lib/
tree-sitter-lua /usr/lib/libtree-sitter-lua.so
tree-sitter-lua /usr/lib/libtree-sitter-lua.so.0
tree-sitter-lua /usr/lib/libtree-sitter-lua.so.0.1
tree-sitter-lua /usr/lib/pkgconfig/
tree-sitter-lua /usr/lib/pkgconfig/tree-sitter-lua.pc
tree-sitter-lua /usr/lib/tree_sitter/
tree-sitter-lua /usr/lib/tree_sitter/lua.so
tree-sitter-lua /usr/share/
tree-sitter-lua /usr/share/doc/
tree-sitter-lua /usr/share/doc/tree-sitter-lua/
tree-sitter-lua /usr/share/doc/tree-sitter-lua/README.md
tree-sitter-lua /usr/share/licenses/
tree-sitter-lua /usr/share/licenses/tree-sitter-lua/
tree-sitter-lua /usr/share/licenses/tree-sitter-lua/LICENSE.md
```

</p>
</details> 

- `/usr/lib/libtree-sitter-lua.so` is a symlink to `/usr/lib/libtree-sitter-lua.so.0`
- `/usr/lib/libtree-sitter-lua.so.0` is a symlink to `/usr/lib/libtree-sitter-lua.so.0.1`
We can probably skip doing that and just do `.so` or `.so.${MAJOR_VERSION}`
- LICENSE is already handled.
- We can decide if we wanna ship the README's for the parsers.
- I'm not sure about if we want to do `/usr/lib/tree_sitter/${filetype}.so`
(`/usr/lib/tree_sitter/lua.so` is a symlink to `/usr/lib/libtree-sitter-lua.so`)
- We probably wanna ship the header and pkgconfig files.
(`/usr/include/tree_sitter/tree-sitter-lua.h`)
(`/usr/lib/pkgconfig/tree-sitter-lua.pc`)

Reply:
ok, I shall change the build scripts to follow ArchLinux's style.

Reply:
Since we seem to be following that model, that seems like the most sensible choice.